Source/Core/Core/HW/EXI_DeviceSlippi.cpp

```diff
@@ -1061,8 +1061,21 @@ void CEXISlippi::prepareFrameData(u8 *payload)
 
 	// If loading from queue, move on to the next replay if we have past endFrame
 	auto watchSettings = replayComm->current;
+	if (frameIndex == watchSettings.startFrame)
+	{
+		std::cout << "[GAME_START]" << std::endl;
+	}
+
+	// This is exactly where the Game! text shows, not when the game really finishes.
+	// Should actually wait 114 frames to send the signal in order to be completely accurate.
+	if (frameIndex == watchSettings.endFrame - 124)
```

Comment:
Why are you subtracting 124? In my experience, if endFrame is set to lastFrame as given by Slippi, that is exactly where the "Game!" text shows.

Reply:
To be honest, I don't remember the exact reason, it's been too long. I think it had to do with the fact that the game actually starts at frame -123 (+1 frame because some reason). @NikhilNarayana said he's been using this code but had to modify it, so this PR might be obsolete a this point.

Reply:
https://github.com/project-slippi/Ishiiruka/tree/feature/cout

yeah I started with your code and changed the setup after some testing. I would close this PR and I'll make a new PR with that branch.